app/src/main/java/org/jellyfin/androidtv/JellyfinApplication.kt

```diff
@@ -61,4 +61,8 @@ class JellyfinApplication : Application() {
 
 		TelemetryService.init(this)
 	}
+	/**
+	 * androidx.work causes a startup crash when it returns true
+	 */
+	override fun isDeviceProtectedStorage() = false
```

Comment:
I don't think an app is supposed to overwrite this variable. Please elaborate on why this is causing an issue and why this change is required.

Reply:
1. On my device, isDeviceProtectedStorage always returns true, and WorkManager throws an exception directly, causing the application to fail to start.
2. Due to WorkManager, the App won't crash at startup only when isDeviceProtectedStorage returns true. So when I return false directly, the App won't get worse in any case, and this can give users like me a chance to use the App normally.
3. After I modified the return value, the App can start normally and its functions are normal.

Reply:
https://issuetracker.google.com/issues/112773820?pli=1

Reply:
The app does not have directBootAware set so the linked issue should be irrelevant.